apps/cli/src/lib/title-text.ts

```diff
@@ -0,0 +1,7 @@
+// https://manytools.org/hacker-tools/ascii-banner/
+export const titleText = `
+ ___         _                ___                   _    ___ _    ___
```

Comment:
We seem to be missing parts of the CLI

Reply:
![image](https://github.com/arkahna/featureboard-sdks/assets/453152/b5b0c480-a308-48b1-9627-29bc1e92d7a1)

I removed the version